tf2_perception_msgs/src/tf2_perception_msgs/tf2_perception_msgs.py

```diff
@@ -0,0 +1,140 @@
+from geometry_msgs.msg import TransformStamped, PoseWithCovarianceStamped
+
+from perception_msgs.msg import ObjectState, Object, ObjectList, EgoData
+from perception_msgs_utils.convenience_state_getters import get_pose_with_covariance
+from perception_msgs_utils.convenience_state_setters import set_pose_with_covariance_from_gm_pose_with_covariance
+from std_msgs.msg import Header
+import tf2_ros
+import tf2_geometry_msgs
+
+
+def to_msg_msg(msg):
+    return msg
+
+tf2_ros.ConvertRegistration().add_to_msg(ObjectState, to_msg_msg)
+tf2_ros.ConvertRegistration().add_to_msg(Object, to_msg_msg)
+tf2_ros.ConvertRegistration().add_to_msg(ObjectList, to_msg_msg)
+tf2_ros.ConvertRegistration().add_to_msg(EgoData, to_msg_msg)
+
+
+def from_msg_msg(msg):
+    return msg
+
+tf2_ros.ConvertRegistration().add_from_msg(ObjectState, from_msg_msg)
+tf2_ros.ConvertRegistration().add_from_msg(Object, from_msg_msg)
+tf2_ros.ConvertRegistration().add_from_msg(ObjectList, from_msg_msg)
+tf2_ros.ConvertRegistration().add_from_msg(EgoData, from_msg_msg)
+
+def do_transform_state(
+        state: ObjectState,
+        transform: TransformStamped) -> ObjectState:
+    """
+    Apply a `Transform` or `TransformStamped` on a `ObjectState`.
```

Comment:
```suggestion
    Apply a `Transform` or `TransformStamped` to `ObjectState`.
```
Is `Transform` really supported or only `TransformStamped`?